src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
Oh wait, I just remember why I didn't check full page but a row of the table only. It's because regex comparison takes too long when the wildcard represents many lines of code. Apparently, the test driver will hang very long until the comparison finishes.